tools/run-integration-test.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+# OSDFIR Infrastructure integration tests
+# This script can be used to test the integration between dfTimewolf, Timesketch, and Turbinia
+# Requirements:
+# - have 'kubectl', 'jq' and 'expect' packages installed
+# - have the OSDFIR Infrastructure Helm chart deployed and are authenticated to the GKE cluster
+# - have the 'dfTimewolf', 'turbinia-client', and 'timesketch' CLI's installed
+
+set -o posix
+set -e
+
+RELEASE="test"
+DISK="disk-1"
+SKETCH_ID="1"
+FAILED=0
+DATE=$(date -I)
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide GCP project and zone"
+  echo "$0 [PROJECT] [ZONE]"
+  exit 1
+fi
+
+GCP_PROJECT="$1"
+GCP_ZONE="$2"
+
+echo -n "Started at "
+date -Iseconds
+
+# Forward k8s services
+echo "Forwarding k8s $RELEASE services"
+kubectl --namespace default port-forward service/$RELEASE-timesketch 5000:5000 > /dev/null 2>&1 &
+kubectl --namespace default port-forward service/$RELEASE-turbinia 8000:8000  > /dev/null 2>&1 &
+
+# Back up existing Timesketch configs else script will attempt to connect to wrong Timesketch instance
+if  [ -f ~/.timesketchrc ] && [ -f ~/.timesketch.token ] 
+then
+    echo "Backing up existing Timesketch configs to ~/.timesketchrc.$DATE and ~/.timesketch.token.$DATE"
```

Comment:
Do we need to keep these?

Reply:
Ehh not really given only should be run within Jenkins anyways. This is more for admin testings/fixes and not having to manually remove the configs each time but I can consider just removing it overall as tests stabilize